game-using-jsDOM/script.js

```diff
@@ -0,0 +1,219 @@
+// Array of words that we will use for the game 
+const words = ['penguins', 'elephant', 'koala', 'synonym', 'peanut',
+    'english', 'ruby', 'react', 'friday', 'airplane', 'banana', 'birthday',
+    'waterfall', 'trampoline', 'window', 'president', 'basketball', 'hiccup',
+    'cheerleader', 'kangaroo', 'groundhog', 'soccer', 'handball', 'caterpillar',
+    'shoelaces', 'hungry', 'fishing', 'monster'];
+
+/**
+ * Gets a random word from the words array declared on line 2
+ * @returns a word that has been randomly selected from the words_array
+ */
+function getWord() {
+    return (words[Math.floor(Math.random() * words.length)]);
+}
+
+// grab and store the body element tag
+let body = document.getElementsByTagName("body")[0];
+
+/**
+ * Creates new elements of the type passed by the parameter elementType
+ * Updates elements innerHTML to be text arg and sets class name 
+ * @param {string} elementType 
+ * @param {string} text 
+ * @param {string} className 
+ * @returns the created element 
+ */
+function generateElements(elementType, text, className) {
+    let element = document.createElement(elementType);
+    element.innerHTML = text;
+    element.className = className;
+    return element;
+}
+
+/**
+ * Creates a new button element and modifies the button text 
+ * to the buttonName arg and sets a class attribute 
+ * @param {string} buttonName 
+ * @param {string} className 
+ * @returns the created button 
+ */
+function generateButtons(buttonName, className) {
```

Comment:
Same as `generateElements`

Reply:
Thank you for pointing this out! I combined the two into a single function now